Makefile

```diff
@@ -53,7 +51,7 @@ test1: clean clean-test-env setup-test-env
 	@echo " ="
 	@. ${TEST_ENV}/bin/activate           && \
 		${SETUP}           				  && \
-		nodeenv -j 4 nodeenv
+		nodeenv -j 4 nodeenv-env
```

Comment:
having a directory called `nodenv` next to a Python file called `nodeenv.py` confuses things, as both are considered Python packages. This make sure not tool can be confused